SubmissionExample.ipynb

```diff
@@ -457,46 +370,13 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 37,
+   "execution_count": null,
    "metadata": {},
-   "outputs": [
-    {
-     "name": "stdout",
-     "output_type": "stream",
-     "text": [
-      "-- SingleStream Accuracy\n",
-      "accuracy=71.676%, good=35838, total=50000\n",
-      "\n",
-      "-- SingleStream Summary\n",
-      "================================================\n",
-      "MLPerf Results Summary\n",
-      "================================================\n",
-      "SUT name : PySUT\n",
-      "Scenario : Single Stream\n",
-      "Mode     : Performance\n",
-      "90th percentile latency (ns) : 3405359\n",
-      "Result is : VALID\n",
-      "  Min duration satisfied : Yes\n",
-      "  Min queries satisfied : Yes\n",
-      "\n",
-      "-- Server Summary\n",
-      "================================================\n",
-      "MLPerf Results Summary\n",
-      "================================================\n",
-      "SUT name : PySUT\n",
-      "Scenario : Server\n",
-      "Mode     : Performance\n",
-      "Scheduled samples per second : 145.35\n",
-      "Result is : VALID\n",
-      "  Performance constraints satisfied : Yes\n",
-      "  Min duration satisfied : Yes\n"
-     ]
-    }
-   ],
+   "outputs": [],
```

Comment:
show part of the output ?

Reply:
Same as above.

Reply:
Added output to the later-stages (long-runtime) for reference without running.